src/Kernel32.Shared/Kernel32Extensions.cs

```diff
@@ -35,8 +36,9 @@ public static unsafe string GetMessage(this Win32ErrorCode error)
         /// Throws an exception when an error occurs.
         /// </summary>
         /// <param name="errorCode">The result of the P/Invoke call.</param>
+        /// <param name="dwLanguageId">The languagte identifier for the message</param>
         /// <exception cref="Win32Exception">If <paramref name="errorCode"/> is not <see cref="Win32ErrorCode.ERROR_SUCCESS"/></exception>
-        public static void ThrowOnError(this Win32ErrorCode errorCode)
+        public static void ThrowOnError(this Win32ErrorCode errorCode, int dwLanguageId = 0)
```

Comment:
Although you added the parameter, you aren't propagating it to the ctor of `Win32Exception`